services/api/Tweek.ApiService.NetCore/Controllers/ContextController.cs

```diff
@@ -30,9 +42,18 @@ public ContextController(IContextDriver contextDriver, JsonSerializer serializer
         {
             if (!_checkAccess(User, new Identity(identityType, identityId))) return Forbid();
             if (data.Count == 0) return Ok();
-
             var identity = new Identity(identityType, identityId);
-            await _contextDriver.AppendContext(identity, data);
+
+            try{
+                await _contextDriver.AppendContext(identity, data);
+            }catch(ArgumentException e)
+            {
+                _logger.LogWarning(
```

Comment:
But what about the reason for the failure? It will be lost..

Reply:
For the reason, in the message, we should add which argument caused the problem and. (since it's 400 error).
A logging middleware should be able to read this data.

Reply:
Ok, I'll give that a try.

Reply:
I think, that since 400 errors are the client fault, we should provide him information about what was invalid. (maybe in the future we'll need better error formatting).
And if the client continues to misbehave, we can use a rate-limiting layer.

Reply:
Yes, but we need to be careful. An error message can provide too much information - so we need to find the way to give the user as much information as we can, and still keeping us secure. For example, a malicious user can use this feature to learn about the scheme.
I am thinking to return something like `prop <> is invalid according to type scheme`.